cmd/ulxly/bridge_service/agglayer/error.go

```diff
@@ -0,0 +1,5 @@
+package agglayer
+
+type errorResponse struct {
```

Comment:
I guess I would typically expect that a custom error implemented the error interface `Error() string` but up to you